src/routes/Edit/VideoSplitter.tsx

```diff
@@ -404,7 +404,7 @@ const VideoSplitter = ({
                 </SegmentUI.Group>
               </SegmentUI>
               <Button.Group attached="bottom">
-                <Button onClick={addSegment}>
+                <Button onClick={addSegment} color="blue">
```

Comment:
I think this looks nice, my only worry is that it does not match any of the current style (i.e. nothing else is "blue").

I'm not too worried about this however since we will need to update the style to match the brand and identity decisions later.

Reply:
Yes, the buttons on this list do not follow the teal/grey color scheme. It is RGB. Anyhow we can make everything teal/grey. Please let me know if you like this idea.

Reply:
Actually, after looking some more we do use "blue" on several elements, just not any buttons yet. I think this is fine for now. I don't want to stress too much on style right now.

Reply:
You are right. I did some cosmetic changes just to get my hands dirty, to see what works. Now ready for the good stuff. ;)